7.0/bin-compat/anthem

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# "Hack" to avoid crashing tests on legacy Odoo versions
+
+set -e
+echo "WARNING! Legacy project => Anthem will not do anything"
+exit 0
```

Comment:
Do we want to exit 1 so it's a hard failure?

Reply:
I think the tests on travis might fail, thus not pushing the image to the hub, if we have a hard failure here. Not sure, tho.

Reply:
Got it